site/docs/reference/function.md

```diff
@@ -7,15 +7,16 @@ slug: '/function'
 
 <!-- TODO generate this table automatically from the same output as iasql_help -->
```

Comment:
I think we can remove this once https://github.com/iasql/iasql-engine/pull/1624 is in since these are documented there?

Reply:
I actually removed that file from the list, because i saw it was documented on a different part. I could re-add